Artem Yurchenko/README.md

```diff
@@ -0,0 +1,56 @@
+Aida64
+=============
+
+
+[![AIDA64 Extreme](https://i.im.ge/2023/05/25/FMXWAx.png)](https://www.aida64.com/downloads/ZjIxYzBjZTM=) 
+[![Google Play](https://i.im.ge/2023/05/25/FMXV1J.png)](https://play.google.com/store/apps/details?id=com.finalwire.aida64&hl=en&pli=1) 
+[![App Store](https://i.im.ge/2023/05/25/FMXZpa.png)](https://apps.apple.com/app/apple-store/id979579523) 
+[![Ubuntu](https://i.im.ge/2023/05/25/FMXfwy.png)](https://download.aida64.com/mobile/aida64.finalwire_1.0.1_multi.click) 
+
+
+## Downloads & Beta Versions
+We constantly update our products to support the latest hardware and technologies. You can download the latest version of our products, as well as older versions and beta releases. We release regular weekly beta updates for our products to assure our customers always have the latest hardware support and the most extensive hardware database. If you have a technical issue, it is strongly recommended that you first upgrade to the latest beta version before contacting FinalWire for assistance.
+
+## Products
+* [AIDA64 Extreme](https://www.aida64.com/products/aida64-extreme)
+* [AIDA64 Engineer](https://www.aida64.com/products/aida64-engineer)
+* [AIDA64 Business](https://www.aida64.com/products/aida64-business)
+* [AIDA64 Network Audits](https://www.aida64.com/products/aida64-network-audit)
+* [AIDA64 for Mobile devices](https://www.aida64.com/mobile-applications)
+
+
+## Updating to latest version
+In case you have a valid license, you can upgrade to a new stable or beta release, as long as your maintenance period has not yet expired. You can renew your maintenance period any time on our [License Renewal](https://www.aida64.com/support/renewal) page.
+You can learn more about AIDA64 licensing on our [Licensing page](https://www.aida64.com/licensing).
+
+
+## Frequently Asked Questions
+For the past nearly twenty years, we have received a large number of questions which have been asked time and again by our users. We have collected these, together with our answers, in our Knowledge Base, where you will find answers to the most common questions related to our products, potential technical issues, hardware and software support, licensing and payment methods.
+
+[Browse FAQ >](https://www.aida64.com/support/faq)
+
+
+## Why is my AIDA64 product key not working?
+AIDA64 product keys are uniquely generated based on the software edition, software version and other license parameters. The following cases are most common issues that prevent the software key from working properly.
```

Comment:
'The following cases are THE most common issues that prevent the software key from working properly.' (missing 'the').